src/Sulu/Bundle/PageBundle/Resources/js/views/Form/toolbarActions/TemplateToolbarAction.js

```diff
@@ -26,6 +27,23 @@ export default class TemplateToolbarAction extends AbstractFormToolbarAction {
             this.resourceFormStore.setType(this.defaultTemplate);
         }
 
+        const parentPageId = this.router.attributes.parentId;
+        if (parentPageId && this.webspace && !this.parentPage) {
+            ResourceRequester.get('pages', {
+                id: parentPageId,
+                language: this.router.attributes.locale,
+            }).then((response) => {
+                this.parentPage = response;
+                const parentTemplate = response.template;
+                const defaultTemplate = this.webspace.defaultTemplates['page'].find((defaultTemplate) => {
+                    return defaultTemplate.parentTemplate === parentTemplate;
+                });
+                if (defaultTemplate) {
```

Comment:
Would add an empty line before this one.

Reply:
Ok